ftw/contacts/tests/test_utils_safe_html.py

```diff
@@ -0,0 +1,32 @@
+from ftw.contacts.testing import FTW_CONTACTS_INTEGRATION_TESTING
+from ftw.contacts.utils import safe_html
+from plone.app.testing import setRoles
+from plone.app.testing import TEST_USER_ID
+from unittest2 import TestCase
+
+
+class TestSafeHtml(TestCase):
+
+    layer = FTW_CONTACTS_INTEGRATION_TESTING
```

Comment:
By replacing that with functional testing and drop the integration testing layer altogether, which will make running all the tests probably ~ 20 sec faster :wink: